website/citations/providers.py

```diff
@@ -138,7 +138,7 @@ def set_config(self, node_addon, user, external_list_id, external_list_name, aut
     def add_user_auth(self, node_addon, user, external_account_id):
         """Adds authorization to a node
         if the user has authorization to grant"""
-        external_account = ExternalAccount.load(external_account_id)
+        external_account = ExternalAccount.objects.get(_id=external_account_id)
```

Comment:
This will throw `ExternalAccount.DoesNotExist()` if it's not there. MODM's load didn't used to do that. If you're looking for the same behavior you can do `.filter().first()` and the retval will be `None` if the criteria isn't met. (h/t @chrisseto This is also slower because it implies a sort but ... we can fix that later.)